package.json

```diff
@@ -28,7 +28,7 @@
     "@blackbaud/skyux-lib-clipboard": "^4.0.0",
     "@blackbaud/skyux-lib-media": "^4.0.0",
     "@blackbaud/skyux-lib-restricted-view": "^4.0.0",
-    "@blackbaud/skyux-lib-stache": "^4.0.0",
+    "@blackbaud/skyux-lib-stache": "^4.1.0",
```

Comment:
☝️ Adds `restricted` support for the left-hand nav menu: https://github.com/blackbaud/skyux-lib-stache/pull/81